### ISSUE TITLE
How to encode the place of an inline addition?

### ISSUE THREAD

**Issue by ppasedach** [chchch/upama#29]
In the middle section of this folio, in line 3 we have an eye-skip, which the scribe himself became aware of, such that he added an insertion mark, and the missing part of the text in the lowest line of the folio, after which the text continues:

![Selection_235](https://user-images.githubusercontent.com/11517856/143495851-ca995241-c088-45a0-a300-5d6645076f0e.png)

So verses 35 and 36 would be something like

```
<lg type="verse" xml:id="HV_13_35">
<l>ketuṃ yaśāṃsi samarāpa<space type="binding-hole"/>ṇiṣu pragalbham</l>
<l>iṃduprabhādhavala<!--XXX arrows, see HV_13_48 XXX--><add place="l.7">bhāṃsi kṛtaspṛhāṇāt|</add></l>
<l><add place="l.7">ceto dunoti bhava<g rend="deleted daṇḍa">¦</g><space type="binding hole"/>tām anutāpahetuḥ</add></l>
<l><add place="l.7">kasmāt pareṇa kuta eṣa na mānabhaṃgaḥ||</add></l>
</lg>
<lg type="verse" xml:id="HV_13_36">
<l><add place="l.7">saṃhārakālaśikhitīvrataro 'pi manyu</add></l>
<l><add place="l.7">ruttaṃsalūniphala</add> eva sa eṣa jātaḥ|</l>
<l>cchinnā itīva bhavataḥ kiraṇacchaṭābhir</l>
<l>ete hasaṃti maṇikuṭṭimasīmni hārāḥ||</l>
</lg>
```
and in verse 48 
```
<lg type="verse" xml:id="HV_13_48">
<l>gaṃbhīraśabditaravaśravaṇotthabhīti</l>
<l>na syatturaṃgakakulasya karālamūrtteḥ|</l>
<l>lābho 'sti kaḥ kaṭhinakuṃjaracakravāla-</l>
<l>kuṃbhasthalīvidalanād aparo <lb n="7"/><note place="inline">3 X [text missing in 35-36] X 3 ꣹</add>XXX-->mṛgāreḥ|</l>
```
But ` [text missing in 35-36]` is not really XML. Maybe one would give the `<add>`s in 35-36 `@xml:id`s, and list them in one way or another in the note in 48? Not sure. Probably better to have an easier solution.

**Comment by ppasedach** [comment]
[The Sanskrit is a very raw transcript here...]

**Comment by chchch** [comment]
Wow that scribe really messed up aligning the text on either side of the stringhole.

What about
```xml
<lg type="verse" xml:id="HV_13_35">
  <l>ketuṃ yaśāṃsi samarāpa<space type="binding-hole"/>ṇiṣu pragalbham</l>
  <l>iṃduprabhādhavala<!--XXX arrows, see HV_13_48 XXX--><add n="l.7a"><note place="inline">3 X</note>bhāṃsi kṛtaspṛhāṇāt|</add></l>
  <l><add n="l.7b">eto dunoti bhava<g rend="deleted daṇḍa">¦</g><space type="binding hole"/>tām anutāpahetuḥ</add></l>
  <l><add n="l.7c">kasmāt pareṇa kuta eṣa na mānabhaṃgaḥ||</add></l>
</lg>
<lg type="verse" xml:id="HV_13_36">
  <l><add n="l.7d">saṃhārakālaśikhitīvrataro 'pi manyu</add></l>
  <l><add n="l.7e">ruttaṃsalūniphala <note place="inline">X 3 ꣹</note></add> eva sa eṣa jātaḥ|</l>
  <l>cchinnā itīva bhavataḥ kiraṇacchaṭābhir</l>
  <l>ete hasaṃti maṇikuṭṭimasīmni hārāḥ||</l>
</lg>
```

Then
```xml
<l>kuṃbhasthalīvidalanād aparo <lb n="7"/><ptr corresp="#l.7a #l.7b #l.7c #l.7d #l.7e"/>mṛgāreḥ|</l>
```

**Comment by MAMMHOTH** [comment]
Hello, Maxi here, I'm the student who's currently working on this MS. I have one question:
Would it not make sense to use metamark instead of note here?
A similar problem was discussed in Issue #12. It was about the same MS where we had inline arrows and the scribe added the notes in the above and below margins with Xs and numbers attached. There it was decided to encode the X and the number denoting the line where it should be added as, e.g.: <metamark place="above and below">^</metamark><add place="bottom-margin"><metamark>✗</metamark>pratha<metamark>7</metamark></add> 

Could we here, thus, make it look like this?:
> 
> ```
> <lg type="verse" xml:id="HV_13_35">
>   <l>ketuṃ yaśāṃsi samarāpa<space type="binding-hole"/>ṇiṣu pragalbham</l>
>   <l>iṃduprabhādhavala<metamark place="above and below">^</metamark><add n="l.7a"><metamark place="inline">3 X</metamark>bhāṃsi kṛtaspṛhāṇāt|</add></l>
>   <l><add n="l.7b">eto dunoti bhava<g rend="deleted daṇḍa">¦</g><space type="binding hole"/>tām anutāpahetuḥ</add></l>
>   <l><add n="l.7c">kasmāt pareṇa kuta eṣa na mānabhaṃgaḥ||</add></l>
> </lg>
> <lg type="verse" xml:id="HV_13_36">
>   <l><add n="l.7d">saṃhārakālaśikhitīvrataro 'pi manyu</add></l>
>   <l><add n="l.7e">ruttaṃsalūniphala <metamark place="inline">X 3 ꣹</metamark></add> eva sa eṣa jātaḥ|</l>
>   <l>cchinnā itīva bhavataḥ kiraṇacchaṭābhir</l>
>   <l>ete hasaṃti maṇikuṭṭimasīmni hārāḥ||</l>
> </lg>
> ```
> 
> Then
> 
> ```
> <l>kuṃbhasthalīvidalanād aparo <lb n="7"/><ptr corresp="#l.7a #l.7b #l.7c #l.7d #l.7e"/>mṛgāreḥ|</l>
> ```

Would that work?

Looking forward to your feedback.

**Comment by chchch** [comment]
Yes I think that would be fine. I've been using metamark when I want to specify the function of a mark, otherwise it doesn't make such a big difference, I think. So something like,
```xml
<metamark place="above and below" function="insertion">^</metamark>
```
Then, in an imagined future, you would be able to find all the `<metamark>`s that are used to mark an insertion, and do some research on that.

**Comment by MAMMHOTH** [comment]
Thank you very much for your helpful remark and feedback.
